io/disk/fiotest.py

```diff
@@ -44,11 +45,25 @@ def setUp(self):
         """
         Build 'fio'.
         """
-        tarball = self.fetch_asset('http://brick.kernel.dk/snaps/fio-2.1.10.tar.gz')
+        url = "http://brick.kernel.dk/snaps/fio-2.1.10.tar.gz"
```

Comment:
always good to have the url passed from yaml, in case if url does not work/changed.

url = self.params.get('url', default='http://brick.kernel.dk/snaps/fio-2.1.10.tar.gz')